app/assets/javascripts/Components/Modals/submission_file_upload_modal.jsx

```diff
@@ -57,6 +109,19 @@ class SubmissionFileUploadModal extends React.Component {
                 onChange={this.handleFileUpload}
               />
             </div>
+            {this.state.newFiles.length == 1 ? (
```

Comment:
This should all be part of the helper function as well, so that the only new change here is including `{fileInput}`.